README.md

```diff
@@ -76,52 +76,57 @@ python -m pip install pennylane
 
 ## Docker support
 
-**Docker** support exists for building using **CPU** and **GPU** (Nvidia CUDA
-11.1+) images. [See a more detailed description
-here](https://pennylane.readthedocs.io/en/stable/development/guide/installation.html#docker).
+Docker images are found on the [PennyLane Docker Hub page](https://hub.docker.com/u/pennylaneai), where there is also a detailed description about PennyLane Docker support. [See description here](https://docs.pennylane.ai/projects/lightning/en/stable/dev/docker.html) for more information.
 
 ## Getting started
 
-For an introduction to quantum machine learning, guides and resources are available on
-PennyLane's [quantum machine learning hub](https://pennylane.ai/qml/):
+Get up and running quickly with PennyLane by following our [quickstart guide](https://docs.pennylane.ai/en/stable/introduction/pennylane.html), designed to introduce key features and help you start building quantum circuits right away.
+
+Whether you're exploring quantum machine learning (QML), quantum computing, or quantum chemistry, PennyLane offers a wide range of tools and resources to support your research:
 
 <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/readme/gpu_to_qpu.png" align="right" width="400px">
 
-* [What is quantum machine learning?](https://pennylane.ai/qml/whatisqml)
-* [QML tutorials and demos](https://pennylane.ai/qml/demonstrations)
-* [Frequently asked questions](https://pennylane.ai/faq)
-* [Key concepts of QML](https://pennylane.ai/qml/glossary)
-* [QML videos](https://pennylane.ai/qml/videos)
+### Key Resources:
+
+* [Learn quantum programming](https://pennylane.ai/qml/)
+* [PennyLane Codebook](https://pennylane.ai/codebook/)
+* [Coding challenges](https://pennylane.ai/challenges/)
+* [Demos](https://pennylane.ai/qml/demonstrations.html)
+* [Frequently asked questions](https://pennylane.ai/faq.html)
+* [Key concepts](https://pennylane.ai/qml/glossary.html)
+* [Videos](https://pennylane.ai/qml/videos.html)

```

Comment:
I would re-order these to put the research-heavy ones first:

```suggestion
* [Research-oriented demos](https://pennylane.ai/qml/demonstrations.html)
* [Learn quantum programming](https://pennylane.ai/qml/) with the [PennyLane Codebook](https://pennylane.ai/codebook/) and [Coding challenges](https://pennylane.ai/challenges/)
* [Frequently asked questions](https://pennylane.ai/faq.html)
* [Key concepts](https://pennylane.ai/qml/glossary.html)
* [Videos](https://pennylane.ai/qml/videos.html)
```

Reply:
Thanks for the suggestion, Josh!